server/controllers/users.js

```diff
@@ -353,7 +353,8 @@ module.exports = (app) => {
       .then(groups => groups.map(group => groupData.push(group.info)))
       .then(() => groupObjects.map(groupObject => {
         const group = _.find(groupData, {id: groupObject.id})
-        return _.extend(group, { balance: groupObject.getBalance() })
+        return groupObject.getBalance()
```

Comment:
yes because you mixed a synchronous function `_.extend` with a promise `getBalance()`.